support-frontend/assets/pages/digital-subscription-checkout/components/orderSummary/orderSummary.jsx

```diff
@@ -16,11 +17,14 @@ type PropTypes = {
   image: $Call<GridImageType, GridImg>,
   productPrice: ProductPrice,
   title: string,
+  orderIsAGift?: boolean,
 };
 
 function OrderSummary(props: PropTypes) {
-
-  const priceString = getBillingDescription(props.productPrice, props.billingPeriod);
+  const giftBillingPeriod = props.billingPeriod === 'Annual' ? props.billingPeriod : 'Quarterly';
```

Comment:
Import `Annual` and `Quarterly` from billingPeriods.js

Reply:
👍